app/client/components/delivery/records/deliveryRecordsProblemConfirmation.tsx

```diff
@@ -179,245 +178,62 @@ const DeliveryRecordsProblemConfirmationFC = (
               ${textSans.medium()};
               display: flex;
               flex-wrap: wrap;
+              flex-direction: column;
               justify-content: space-between;
               ${minWidth.tablet} {
+                flex-direction: initial;
                 padding: 0 ${space[5]}px;
               }
-            `}
-          >
-            <div
-              css={css`
-                ${minWidth.tablet} {
-                  min-width: 50%;
-                }
-              `}
-            >
-              <dt
-                css={css`
-                  ${dtCss}
-                `}
-              >
-                Reference:
-              </dt>
-              <dd
-                css={css`
-                  ${ddCss}
-                  ${minWidth.tablet} {
-                    min-width: 12ch;
-                  }
-                `}
-              >
-                {problemReferenceId}
-              </dd>
-            </div>
-            <div
-              css={css`
-                flex-grow: 1;
+              div {
```

Comment:
moved child div styles (inc. tablet breakpoint `min-width`) inside the parent's style definition to avoid repitition